ui/js/directives/treeherder/clonejobs.js

```diff
@@ -127,7 +127,6 @@ treeherder.directive('thCloneJobs', [
         var selectJob = function(job, job_selection_type) {
             var jobKey = getJobMapKey(job);
             var jobEl = $('.' + jobKey);
-
             clickJobCb({}, jobEl, job, job_selection_type);
             scrollToElement(jobEl);
```

Comment:
@wlach , yeah you are right, I was putting the wrong object in the jQuery selector